backstop/tests/header.html

```diff
@@ -235,7 +279,7 @@ <h2>Default</h2>
 
             <!-- Profile dropdown button -->
             <button
-              class="iui-button iui-dropdown iui-borderless"
+              class="iui-button iui-borderless"
```

Comment:
why removed?

Reply:
It doesn't appear to be applying any styling.

Reply:
oh ,itneresting.. I think we expected to have an arrow there, to show its a dropdown menu, when yo uclick on it..
but ok, it wasnt there before, so no need.